### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by icarius**
Automating ENV FIVEM_NUM in the builder

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,6 +18,9 @@ RUN wget -O- https://runtime.fivem.net/artifacts/fivem/build_proot_linux/master/
     \
  && apk -p $PWD add tini
 
+RUN FIVEM_NUM=$(echo "${FIVEM_VER}" | cut -d '-' -f 1)
+ENV FIVEM_NUM=${FIVEM_NUM}
```
Comment (frebib) [inline Dockerfile:22]: This breaks the labelling on line 40/42:
`"org.label-schema.version": ""`
Is there any way we can do this in the Dockefile without the `RUN` line to keep that?

**Review by frebib — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (icarius) [reply Dockerfile:22]: Strange it's working for me.
No you can only use RUN on builder

Comment (frebib) [reply Dockerfile:22]: > Strange it's working for me.

If you run `docker inspect` on the built image do you see the label?

Comment (icarius) [reply Dockerfile:22]: image
You're right, it doesn't register

**Comment by icarius** [comment]
Doesn't work sorry

**Pull request closed without merging**